Source/Scene/Camera.js

```diff
@@ -2552,6 +2552,23 @@ define([
             destination = this.getRectangleCameraCoordinates(destination, scratchFlyToDestination);
         }
 
+        var sscc = this._scene.screenSpaceCameraController;
+        var ellipsoid = this._scene.globe.ellipsoid;
+        var destinationCartographic = Cartographic.fromCartesian(destination);
+
+        // Make sure camera doesn't zoom outside set limits
+        if (defined(sscc.minimumZoomDistance) && defined(sscc.maximumZoomDistance)) {
+            destinationCartographic.height = CesiumMath.clamp(destinationCartographic.height, sscc.minimumZoomDistance, sscc.maximumZoomDistance);
```

Comment:
You only need to check that `sscc` is defined. If it is, `minimumZoomDistance` and `maximumZoomDistance` are assumed to be defined.